packs/classfeatures/ancestors.json

```
@@ -30,6 +30,105 @@
                 "mode": "upgrade",
                 "path": "system.skills.soc.rank",
                 "value": 1
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:trait:cursebound",
+                    {
+                        "or": [
+                            "class:oracle",
+                            "feat:first-revelation"
+                        ]
+                    }
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "predicate": [
+                            {
+                                "not": "self:effect:curse-of-ancestral-meddling"
+                            }
+                        ],
+                        "text": "PF2E.SpecificRule.Oracle.CurseSeverity.NoneToMinor"
+                    },
+                    {
+                        "predicate": [
+                            {
+                                "not": "self:effect:curse-of-ancestral-meddling"
+                            }
+                        ],
+                        "text": "@UUID[Compendium.pf2e.feat-effects.Item.Effect: Curse of Ancestral Meddling]"
+                    },
+                    {
+                        "predicate": [
+                            "oracular-curse:stage:1",
+                            "class:oracle"
+                        ],
+                        "text": "PF2E.SpecificRule.Oracle.CurseSeverity.MinorToModerate"
+                    },
+                    {
+                        "predicate": [
+                            "oracular-curse:stage:2",
+                            "class:oracle",
+                            {
+                                "gte": [
+                                    "self:level",
+                                    11
+                                ]
+                            }
+                        ],
+                        "text": "PF2E.SpecificRule.Oracle.CurseSeverity.ModerateToMajor"
+                    },
+                    {
+                        "predicate": [
+                            "oracular-curse:stage:3",
+                            "class:oracle",
+                            {
+                                "gte": [
+                                    "self:level",
+                                    17
+                                ]
+                            }
```

```suggestion
                            "feature:extreme-curse"
```
More compact to just check for the feature, plus future proof in case Paizo does some utterly bizarre class archetypes in a couple years.

---

We can also drop the class check with this